### PR TITLE
ENG-5114: Add orders api definition.

### PR DESCRIPTION
**Pull request by bancsdani**

### CODE REVIEW

File: assets/openapi.yaml [hunk 1]
```diff
@@ -627,6 +627,104 @@ components:
...
+          type: string
+          format: decimal
+          example: 5
+        extended_hours:
+          type: boolean
+          example: false
+        client_order_id:
+          type: string
+          format: uuid
+          example: eb9e2aaa-f71a-4f51-b5b4-52a6c565dad4
```
Comment (Mark-Pintye) [inline assets/openapi.yaml:691]: This is not necessarily a uuid

Comment (bancsdani) [reply assets/openapi.yaml:691]: but its an example for one isn't it?

File: assets/openapi.yaml [hunk 2]
```diff
@@ -627,6 +627,104 @@ components:
...
+          properties:
+            stop_price:
+              type: string
+              format: decimal
+              example: 3.14
+            limit_price:
+              type: string
+              format: decimal
+              example: 3.14
+        comission:
```
Comment (Mark-Pintye) [inline assets/openapi.yaml:718]: typo: `commission

File: assets/openapi.yaml [hunk 3]
```diff
@@ -911,6 +1009,38 @@ paths:
                   clearing_broker:
                     type: string
                     example: Velox
+/trading/accounts/{account_id}/orders:
+  get:
+    security:
+      - BasicAuth: []
+    summary: Retrieve trading details for an account.
```
Comment (Mark-Pintye) [inline assets/openapi.yaml:1016]: Incorrect

**Review by umitanuki — APPROVED** [review]
Looks great. Thanks for filling this

File: assets/openapi.yaml [hunk 4]
```diff
@@ -911,6 +1022,88 @@ paths:
...
+                type: object
+                properties:
+                  id:
+                    type: string
+                    example: c8f1ef5d-edc0-4f23-9ee4-378f19cb92a4
+                  account_number:
+                    type: string
+                    example: 927584925
+                  status:
+                    $ref: "#/components/schemas/AccountStatus"
```
Comment (Mark-Pintye) [inline assets/openapi.yaml:1102]: copy pasta from createaccount

File: assets/openapi.yaml [hunk 5]
```diff
@@ -911,6 +1022,88 @@ paths:
...
+    get:
+      security:
+        - BasicAuth: []
+      summary: Retrieves a list of orders for the account, filtered by the supplied query parameters.
+      tags:
+        - Trading
+      description: Retrieves a list of orders for the account, filtered by the supplied query parameters.
+      parameters:
+        - $ref: "#/components/parameters/AccountID"
+      requestBody:
```
Comment (Mark-Pintye) [inline assets/openapi.yaml:1035]: corresponding responses are missing

File: assets/openapi.yaml [hunk 6]
```diff
@@ -30,6 +30,19 @@ components:
       schema:
         type: string
         format: uuid
+  responses:
+    NotFound:
+      description: Request body format is invalid or request parameters are invalid
```
Comment (Mark-Pintye) [inline assets/openapi.yaml:35]: descriptions swapped

File: assets/openapi.yaml [hunk 7]
```diff
@@ -627,6 +646,254 @@ components:
...
+    OrderObject:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+          example: "61e69015-8549-4bfd-b9c3-01e75843f47d"
+        client_order_id:
+            type: string
+            format: uuid
```
Comment (Mark-Pintye) [inline assets/openapi.yaml:754]: this is still uuid

File: assets/openapi.yaml [hunk 8]
```diff
@@ -911,6 +1178,103 @@ paths:
...
+                type: object
+                properties:
+                  id:
+                    type: string
+                    example: c8f1ef5d-edc0-4f23-9ee4-378f19cb92a4
+                  account_number:
+                    type: string
+                    example: 927584925
+                  status:
+                    $ref: "#/components/schemas/OrderObject"
```
Comment (Mark-Pintye) [inline assets/openapi.yaml:1271]: it does not give back an object with id,account_number,status, just the orderentity

**Review by Mark-Pintye — APPROVED** [review]
if it works generating a request and getting a good response then it looks good

**Review by mahmoudksherif — APPROVED** [review]

**Pull request merged**